### ISSUE TITLE
Do JS minifiers support removing dead code?

### ISSUE THREAD

**Issue by petrosmm** [Taritsyn/WebMarkupMin#76]
In various projects I have worked on, developers leave a lot of dead code in the comments mostly for debugging purposes, however that code get's transported... Does WebMarkupMin and it's accompanying JS minifiers support removing dead code like what is shown below:
```
function prospectusChanged(obj, windowname) {
    // dead code
    if (false)
        alert("optionchanged");

    var URL = document.getElementById(obj).value;
    var _winName = "WindowName";
    if (windowname) _winName = windowname;

    // dead code
    if (false)
       {        alert("optionchanged");
       }

    if (URL !== "") {
        window.open(URL, _winName);
        document.getElementById(obj).selectedIndex = 0;
        return;
    }
}
```

**Comment by Taritsyn** [comment]
Hello!

Have you tried to install the [WebMarkupMin.MsAjax](https://www.nuget.org/packages/WebMarkupMin.MsAjax/) and [WebMarkupMin.NUglify](https://www.nuget.org/packages/WebMarkupMin.NUglify/) modules?

**Comment by petrosmm** [comment]
I have WebMarkupMin.MsAjax however when I looked around I did not become aware that it can remove dead code however I mixed up WebMarkupMin.NUglify with BundleTransformer.NUglify... I will look into that immediately!

**Comment by Taritsyn** [comment]
It seems, that the NUglify JS minifier does not consider such a code to be dead (even with the `RemoveUnneededCode` option). You can [write about this problem](https://github.com/xoofx/NUglify/issues/new) to author of the NUglify library.

**Comment by petersmm** [comment]
Closed with answer: [Microsoft/ajaxmin#3](https://github.com/Microsoft/ajaxmin/issues/3)
